lib/reporter/logger.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var _ = require('lodash');
+var chalk = require('chalk');
+
+var util = require('./util');
+
+function logModule(log, logType, module) {
+  log[logType](chalk.yellow('module name:'), module.name);
+  log[logType](chalk.yellow('version:'), module.version);
+  if (module.error) {
+    log[logType]('error', module.error.message);
```

Comment:
The logger already wraps the title for an error in red
https://github.com/TheAlphaNerd/citgm/blob/reporter/lib/out.js#L65-L67
this is simply passing the error.message